newIDE/app/src/GameDashboard/GameDashboardCard.js

```diff
@@ -0,0 +1,657 @@
+            if (actions.length > 0) {
+              actions.push({
+                type: 'separator',
+              });
+            }
+
+            const file = projectsList[0];
+            actions.push({
+              label: i18n._(t`Delete`),
+              click: async () => {
+                // Extract word translation to ensure it is not wrongly translated in the sentence.
+                const translatedConfirmText = i18n._(t`delete`);
+
+                const answer = await showDeleteConfirmation({
+                  title: t`Delete game`,
+                  message: t`Your game will be deleted. This action is irreversible. Do you want to continue?`,
+                  confirmButtonLabel: t`Delete game`,
+                  fieldMessage: t`To confirm, type "${translatedConfirmText}"`,
+                  confirmText: translatedConfirmText,
+                });
+                if (!answer) return;
+
+                // If the game is registered, unregister it.
+                // If it fails, this will throw, to prevent deleting a game with leaderboards or not owned.
+                if (game) {
+                  try {
+                    await onUnregisterGame();
+                  } catch (error) {
+                    console.error('Unable to unregister the game.', error);
+                    // Alert is handled by onUnregisterGame. Just ensure we don't continue.
+                    return;
+                  }
+                }
+
+                // If there is a project file (local or cloud), remove it.
+                if (file) {
+                  if (file.storageProviderName === 'Cloud') {
+                    await onDeleteCloudProject(file);
+                  } else {
+                    await removeRecentProjectFile(file);
+                  }
+                }
+
+                await onRefreshGames();
+              },
+            });
+          }
+
+          return actions;
+        }}
+      />
+    );
+  };
+
+  const onManageGame = React.useCallback(
+    async () => {
+      if (game) {
+        onOpenGameManager({ game });
+        return;
+      } else {
+        if (!authenticatedUser.profile) {
+          authenticatedUser.onOpenLoginDialog();
+          return;
+        }
+        const answer = await showConfirmation({
+          title: t`Manage game online`,
+          message: t`This game is not registered online. Do you want to register it to access the online features?`,
+          confirmButtonLabel: t`Continue`,
+        });
+        if (!answer) return;
+
+        const registeredGame = await onRegisterProject(projectsList[0]);
+        if (!registeredGame) return;
+
+        await onRefreshGames();
+        onOpenGameManager({ game: registeredGame });
+      }
+    },
+    [
+      game,
+      onOpenGameManager,
+      showConfirmation,
+      onRegisterProject,
+      projectsList,
+      onRefreshGames,
+      authenticatedUser,
```

Comment:
Having the whole `authenticatedUser` as a dependency is scary to me, it can change a lot, redefining the callback (and rendering the components that use it) a lot